treeherder/model/migrations/0055_autoclassify_status.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0054_duplicate_failure_classifications'),
+    ]
+
+    operations = [migrations.RunSQL(
+        sql="""
+BEGIN;
```

Comment:
Would you mind using the list form of RunSQL rather than a string, since the latter has to be parsed by sqlparse, which I'd just managed to remove our use of (I need to update the Django docs to make the list form more obvious).

eg:
https://github.com/mozilla/treeherder/blob/5318924e00157ac8dc76351d527a970a4f6db26f/treeherder/model/migrations/0001_squashed_0053_add_job_platform_option_push_index.py#L725-L728